Lib/fontTools/misc/psCharStrings.py

```diff
@@ -1042,6 +1043,10 @@ def compile(self):
 			raise
 		self.setBytecode(bytecode)
 
+		if self.isCFF2:
+			if self.bytecode and (byteord(self.bytecode[-1]) in (11, 14)):
+				self.bytecode = self.bytecode[:-1]
```

Comment:
As I mentioned earlier, I don't think we should do this transparently.  We should have methods for converting from CFF1 to CFF2 that take care of these, instead of doing them automatically.  I'm fine if you fix this later though.